test/webthreestubclient.h

```diff
@@ -12,6 +12,16 @@ class WebThreeStubClient : public jsonrpc::Client
     public:
         WebThreeStubClient(jsonrpc::IClientConnector &conn) : jsonrpc::Client(conn) {}
 
+        std::string web3_sha3(const std::string& param1) throw (jsonrpc::JsonRpcException)
```

Comment:
throw as exception specification is deprecated. See here for more infos: http://stackoverflow.com/questions/13841559/deprecated-throw-list-in-c11
I would just add a comment.